miden/Cargo.toml

```diff
@@ -32,10 +32,16 @@ harness = false
 name = "program_execution"
 harness = false
 
+[[test]]
+name = "miden"
+path = "tests/integration/main.rs"
+required-features = ["internals"]
+
 [features]
 concurrent = ["prover/concurrent", "std"]
 default = ["std"]
-executable = ["env_logger", "hex/std", "std", "serde/std", "serde_derive", "serde_json/std", "structopt", "rustyline"]
+executable = ["env_logger", "hex/std", "internals", "std", "serde/std", "serde_derive", "serde_json/std", "structopt", "rustyline"]
```

Comment:
It's not immediately obvious why we would need `executable`, that's the endpoint of the workspace, to depend on `internals`. I believe `internals` are meant only for non-production stuff.

Is there any way to circumvent this?

Reply:
the [repl](https://github.com/0xPolygonMiden/miden-vm/blob/next/miden/src/repl/mod.rs#L273) has a dependency on the `Process` struct.  But I agree we should see if we can remove it.

Reply:
Let's create an issue for this so that we don't forget to fix it later.